README.md

```diff
@@ -176,12 +176,6 @@ you to manage and maintain those images and containers in a production environme
 familiar container cli commands.  For more details, see the
 [Container Tools Guide](https://github.com/containers/buildah/tree/master/docs/containertools).
 
-## Podman Legacy API (Varlink)
```

Comment:
We should leave this in with a pointer to the last supported release

Reply:
Added back with a rewording.